maintenance/README.md

```diff
@@ -33,24 +33,52 @@ For the sake of simplicity all examples will use jcxjin-test.
    Maintenance mode on.
 
    ```
-   oc patch route pims-app-test -n jcxjin-test -p \
+   oc patch route pims-app-test -n 354028-test -p \
        '{ "spec": { "to": { "name": "proxy-caddy" }, "port": { "targetPort": "2015-tcp" }}}'
-   oc patch route proxy-caddy -n jcxjin-test -p \
+   oc patch route proxy-caddy -n 354028-test -p \
        '{ "spec": { "to": { "name": "pims-app-test" }, "port": { "targetPort": "8080-tcp" }}}'
    ```
 
    Maintenance mode off.
 
    ```
-   oc patch route pims-app-test -n jcxjin-test -p \
+   oc patch route pims-app-test -n 354028-test -p \
        '{ "spec": { "to": { "name": "pims-app-test" }, "port": { "targetPort": "8080-tcp" }}}'
-   oc patch route proxy-caddy -n jcxjin-test -p \
+   oc patch route proxy-caddy -n 354028-test -p \
        '{ "spec": { "to": { "name": "proxy-caddy" }, "port": { "targetPort": "2015-tcp" }}}'
    ```
 
 ### Build and Deployment
 
-This application's template has been broken down into build and deploy components.
+This application's template has been broken down into build and deploy components. The following commands will create all required bc/dc/service/route objects, assuming that the defaults used in the maintenance script are still correct(see below to confirm:
+
+```
+./maintenance.sh tools s2i
```

Comment:
I looked at the `maintenance.sh` script, there doesn't appear to be an `s2i` command.

Reply:
the s2i command is new, and is included in the below changes.